sympy/matrices/expressions/tests/test_trace.py

```diff
@@ -83,9 +83,8 @@ def test_Trace_doit_deep_False():
     assert Trace(q).doit(deep=False).arg == q
 
 
-@XFAIL
 def test_trace_constant_factor():
-    # Issue 9052: LHS gives Trace(MatMul(A))
+    # Issue 9052
```

Comment:
The issue title should be left in, it's always nice to see what a test is about instead of having to look it up in GitHub.